internal/handler/wallet/backup.go

```diff
@@ -61,10 +62,20 @@ func (w *walletBackupAccount) Handle(params operations.BackupAccountParams) midd
 	} else {
 		// for private key backup, send the private key to the wails frontend
 		privateKey = wlt.GetPrivKey()
+		publicKey = wlt.GetPupKey()
+	}
+
+	// Create a data object containing the private key and public key
+	data := struct {
+		PrivateKey string `json:"privateKey"`
+		PublicKey  string `json:"publicKey"`
+	}{
```

Comment:
can you declare a type struct at the top of the file? so that you can reuse it in the unit test